hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/protocolPB/RefreshUserMappingsProtocolServerSideTranslatorPB.java

```diff
@@ -32,12 +32,12 @@
 public class RefreshUserMappingsProtocolServerSideTranslatorPB implements RefreshUserMappingsProtocolPB {
 
   private final RefreshUserMappingsProtocol impl;
-  
-  private final static RefreshUserToGroupsMappingsResponseProto 
+
+  protected final static RefreshUserToGroupsMappingsResponseProto 
   VOID_REFRESH_USER_GROUPS_MAPPING_RESPONSE =
       RefreshUserToGroupsMappingsResponseProto.newBuilder().build();
 
-  private final static RefreshSuperUserGroupsConfigurationResponseProto
+  protected final static RefreshSuperUserGroupsConfigurationResponseProto
   VOID_REFRESH_SUPERUSER_GROUPS_CONFIGURATION_RESPONSE = 
```

Comment:
Hi, @hfutatzhanghb thanks for your contribution! It is recommended not to modify common because the pipeline will run for a long time. There are only 2 variables here, you can copy them directly to RouterRefreshUserMappingsProtocolServerSideTranslatorPB

Reply:
@KeeProMise Sir , thanks for reminding.  fixed